app/services/google/sheets.py

```
@@ -0,0 +1,269 @@
+
+def _rows_to_json(rows: List[dict]) -> str:
+    if not rows:
+        return "[]"
+    return json.dumps(rows, ensure_ascii=False)
+
+
+async def fetch_formatted(
+    spreadsheet_id: str,
+    sheet_name: Optional[str] = None,
+    columns: Optional[List[str]] = None,
+    format: str = "markdown_table",
```

Ruff (0.15.17)

[error] Function argument `format` is shadowing a Python builtin

(A002)